docs/appendix/apis/components/generic.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Control your generic component with the generic API"
+linkTitle: "Generic"
+weight: 20
+type: "docs"
+description: "The generic API allows you to give commands to your generic components with code instead of with the graphical interface of the Viam app"
```

Comment:
Whatever you end up doing with these descriptions, I think they should end with periods :D